packages/flutter/lib/src/widgets/framework.dart

```diff
@@ -2838,6 +2857,24 @@ abstract class Element extends DiagnosticableTree implements BuildContext {
     return 0;
   }
 
+  // Return a numeric encoding of the specific `Element` concrete subtype.
+  // This is used to determine if a hot reload modified the type hierarchy of
```

Comment:
nit: this is used in [Element.updateChild] to determine...

Reply:
Done